### PR TITLE
git Less profile page

### PR DESCRIPTION
**Pull request by elasticSzyszko**
this contains less refactor for Profile page, Purchase details page along with components for New Address, New Payment Method.

### CODE REVIEW

**Comment by shaunmaharaj** [comment]
no further comments

File: src/style/style.less [hunk 1]
```diff
@@ -20,7 +20,7 @@
  */
 
 // Remove import of legacy style.css once all the styles have been refactored to individual .less files
-@import './style.css';
+// @import './style.css';
```
[the remark this answers is not in the document]
Comment (elasticSzyszko) [reply src/style/style.less]: pull request 33 already removed. After both these requests are in we just have login modal and register user page.

[the remark this answers is not in the document]
Comment (shaunmaharaj) [reply src/style/style.less]: #33 is not merged yet. If it goes in, you'll need to rebase and this change would overwrite what's on the master.

**Comment by shaunmaharaj** [comment]
Rebased this one for you. Merging now

**Pull request merged**